toolchain/check/testdata/basics/multifile_raw_and_textual_ir.carbon

```diff
@@ -37,6 +37,15 @@ fn B() {}
 // CHECK:STDOUT:     inst+1:          {kind: FunctionDecl, arg0: function0, type: type1}
 // CHECK:STDOUT:     inst+2:          {kind: Return}
 // CHECK:STDOUT:   constant_values:
+// CHECK:STDOUT:     instTypeType:    template instTypeType
```

Comment:
Should the special-casing that `insts` has to exclude builtins be added here?

Reply:
Done.